### ISSUE TITLE
Marine Mammal Species Report

### ISSUE THREAD

**Issue by mmosesOCMP** [orcoastalmgmt/orSeaSketch#12]
species names are not italicized

**Comment by danyocum** [comment]
For now, I'm assuming they're all the scientific name, so its first letter capitalized, everything else lower, everything italicized

**Comment by mmosesOCMP** [comment]
@danyocum Yes. See the explanation I added to [#14](https://github.com/orcoastalmgmt/orSeaSketch/issues/14#issuecomment-637855055)
